fili-core/src/main/java/com/yahoo/bard/webservice/web/ErrorMessageFormat.java

```diff
@@ -73,6 +73,8 @@
     SORT_METRICS_NOT_SORTABLE_FORMAT("Sorting not possible on metric(s) '%s'."),
     SORT_METRICS_UNDEFINED("Metric(s) in sort expression '%s' do not exist."),
 
+    DATE_TIME_SORT_VALUE_INVALID("dateTime must always be the first field in the sort list"),
```

Comment:
Right, and that's my point... the error message indicates (emphasis added):

> dateTime must **always** be the first field

By the error message indicating _always_, it may be confusing. A more accurate error message would remove that confusion.